### ISSUE TITLE
Hello i downloaded Comfyui yestreday, it was working perfectly ,now i cant generate images it show me this ComfyUI_windows_portable_nvidia\ComfyUI_windows_portable>pause 

### ISSUE THREAD

**Issue by ricky47896** [comfyanonymous/ComfyUI#6121]
### Your question

![comfyui problem](https://github.com/user-attachments/assets/9b634a0e-c6d5-4fc4-91f2-1aff854770b8)
Hello i downloaded Comfyui yestreday, it was working perfectly ,now i cant generate images it show me this every time I press the Queue button ComfyUI_windows_portable_nvidia\ComfyUI_windows_portable>pause

**Comment by AxelFooley** [comment]
Same here, i am using the default FLUX dev workflow from [here ](https://comfyanonymous.github.io/ComfyUI_examples/flux/#simple-to-use-fp8-checkpoint-version)that always worked, but now every time i queue a job i get that pause message immediately and comfyui crashes.

**Comment by ULTRA-VAGUE** [comment]
Yes, I can confirm that this is a widespread issue. Same thing with the regular Flux DEV Workflow. it just stops generating. even after a complete reinstall.

**Comment by LukeG89** [comment]
Could it be related to this issue?
- https://github.com/comfyanonymous/ComfyUI/issues/6188

Comfy gave this solution: https://github.com/comfyanonymous/ComfyUI/issues/6188#issuecomment-2560865162

Anyways, you should give more information (full logs, specs, etc.)

**Comment by ULTRA-VAGUE** [comment]
> Could it be related to this issue?
> 
>     * [Hunyuan on 3090 keeps crashing (Reconnecting) #6188](https://github.com/comfyanonymous/ComfyUI/issues/6188)
> 
> 
> Comfy gave this solution: [#6188 (comment)](https://github.com/comfyanonymous/ComfyUI/issues/6188#issuecomment-2560865162)
> 
> Anyways, you should give more information (full logs, specs, etc.)




Full Logs from the Commandline coming up. not that it´ll help much.


`Total VRAM 24564 MB, total RAM 32767 MB
pytorch version: 2.5.1+cu124
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 4090 : cudaMallocAsync
Using pytorch attention
[Prompt Server] web root: C:\Users\user\Desktop\ComfyUI_windows_portable\ComfyUI\web
### Loading: ComfyUI-Manager (V2.55.5)
### ComfyUI Version: v0.3.9-22-g99a1fb6 | Released on '2024-12-24'

Import times for custom nodes:
   0.0 seconds: C:\Users\user\Desktop\ComfyUI_windows_portable\ComfyUI\custom_nodes\websocket_image_save.py
   0.7 seconds: C:\Users\user\Desktop\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager

Starting server

To see the GUI go to: http://0.0.0.0:7860
To see the GUI go to: http://[::]:7860
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/model-list.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/alter-list.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/github-stats.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/custom-node-list.json
[ComfyUI-Manager] default cache updated: https://raw.githubusercontent.com/ltdrdata/ComfyUI-Manager/main/extension-node-map.json
FETCH DATA from: C:\Users\user\Desktop\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\extension-node-map.json [DONE]
FETCH DATA from: C:\Users\user\Desktop\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager\extension-node-map.json [DONE]
got prompt
Using pytorch attention in VAE
Using pytorch attention in VAE`






that's when it just stops working and you get the error described above. it´s literally a fresh install of win 10 with just comfy running flux 1 Dev using the comfyui wiki tutorial.

**Comment by ULTRA-VAGUE** [comment]
Setting the Pagefile to "System Managed Size" worked.
